daemons/compute/Readme.md

```diff
@@ -151,20 +180,51 @@ requests that match both `namespace` and `workflow`.
 | ----- | ---- | ----------- |
 | `uids` | string[] | List of data movement requests associated with the given workflow and namespace. |
 
+## Canceling a data movement request
+
+Data movement requests can be cancelled after creation of the request.
+
+### Cancel Request
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| `workflow` | Workflow | The name and namespace of the initiating workflow |
+| `uid` | string | The unique identifier for the data movement resource to be cancelled |
+
+### Cancel Response
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| `status` | Status | Cancel status of the data movement request |
+| `message` | string | String providing detailed message pertaining to the cancellation of the request |
+
+#### Cancel Response `Status`
+
+| Value | Name | Description |
+| ----- | ---- | ----------- |
+| 0 | Invalid | The cancel request was found to be invalid |
+| 1 | Not Found | The request with the supplied UID was not found |
+| 2 | Success | The data movement request was cancelled successfully |
```

Comment:
Since we will be polling this cancellation request and completion is asynchronous, should there also be an "In Progress" response for when this call is made before the cancellation has completed?

Reply:
Great question. You made me realize that I missed an update to the Status Response. I will add that.

To recap from an API user's perspective, the user will do the following for cancellation:

1. Request a cancellation via Cancel Request and verify that the Cancel Response status is `Success`.
2. Poll the status via Status Request until state is `Completed` and status is `Cancelled`.

The successful response from #1 is an indication that the cancellation is in progress because that means the k8s data movement resource was updated with the cancel flag. We could explore adding another state to indicate that a cancellation in progress.

Reply:
@NateRoiger and I had some discussions and we decided to add a `Cancelling` state for the API. I will add a new state for the Status Response for this and update the PR.